### ISSUE TITLE
Patternfly Icons: incorrect directory

### ISSUE THREAD

**Issue by Roballen** [keycloakify/keycloakify#427]
First, thanks for the great tool and starter project. It's nice to work in a more familiar env.

I am sure I am missing something, but I have been trying to figure this out for a while. In Storybook, everything is working well. In fact on my keycloak-testing-container everything is working fine v21.1.2, but as soon as I try and deploy the jar to another instance v22.0.3 the default patternfly icons appear to be missing, but are in fact in a different directory. How can I resolve?

<img width="611" alt="image" src="https://github.com/keycloakify/keycloakify/assets/187707/bc9ee391-e05f-48a1-82fb-5fce32be8d36">

On this page the theme is looking for the icons at http://0.0.0.0:8080/resources/r9fgj/login/{theme name}/resources_common/node_modules/patternfly/dist/fonts/fontawesome-webfont.woff2?v=4.7.0

When they, in fact, are actually located at http://0.0.0.0:8080/resources/r9fgj/login/resources_common/node_modules/patternfly/dist/fonts/fontawesome-webfont.woff2?v=4.7.0

In short, it appears that it is looking for the icons in a url with the theme name in it, when they exist in a location without the theme name.

To me this implies, that the icons are no bundled with the theme, because when I look in the jar they aren't there, but it is beyond me how to achieve this in a CI environment. Do I just need to manually copy the files to the correct place?

Any pointers in the right directions would be awesome.

**Comment by garronej** [comment]
Hey @Roballen,  

Sorry it took me two days to check this issue out.  
I'll look into it.  
Could you please just fork the starter, make it so that an icon is used in the login page and share the link here?  
Thanks in advance for the reproduction repo!

**Comment by garronej** [comment]
Fixed! Thanks for reporting!

**Comment by Roballen** [comment]
thanks so much, I was just getting around to isolating. I had a fragile workaround but this looks much better

**Comment by com-admin** [comment]
facing related issue in the 8.1.2 keycloakify-starter `The resource from “http://localhost:8080/resources/0vyio/login/keycloakify-starter/resources_common/node_modules/patternfly/dist/css/patternfly-additions.min.css” was blocked due to MIME type (“”) mismatch (X-Content-Type-Options: nosniff)`
not sure if its related to this issue

**Comment by garronej** [comment]
Hi @com-admin,  

I think this is an unrelated issue.
